packages/kolibri-tools/lib/i18n/ExtractStrings.js

```diff
@@ -361,10 +359,11 @@ ExtractStrings.prototype.apply = function(compiler) {
 };
 
 ExtractStrings.prototype.writeOutput = function(messageExport) {
+  const writePath =`${this.messageDir}/locale/CSV_FILES/en/LC_MESSAGES`;
```

Comment:
Is this still being used as a webpack plugin, or has it been separated? I'd much prefer the latter and I think it would make things less complicated overall.

Reply:
Explained below.